backend/python/app/tests/services/test_story_service.py

```diff
@@ -11,23 +14,51 @@ def test_get_story(app, db, services):
     assert_story_equals_model(resp, obj, graphql_response=False)
 
 
-def test_get_story_invalid_id():
-    pass
-
-
-def test_get_stories():
-    pass
+def test_get_story_invalid_id(app, db, services):
+    with pytest.raises(Exception) as e:
+        _ = services["story"].get_story(-1)
+        assert "Invalid id" in str(e.value)
 
+    obj = Story(title="title", description="description", youtube_link="", level=1)
+    db.session.add(obj)
+    assert db.session.commit() == None
 
-def test_create_story():
-    pass
+    with pytest.raises(Exception) as e:
+        _ = services["story"].get_story(obj.id + 1)
+        assert "Invalid id" in str(e.value)
 
 
-def test_create_story_raises_error_if_story_commit_fails_and_nothing_saved():
```

Comment:
git diff came out really weird looking but basically I removed `test_create_story_raises_error_if_story_commit_fails_and_nothing_saved` and `test_create_story_raises_error_if_story_content_commit_fails_and_nothing_saved` because i feel like testing for a broken database connection is not worth the extra time to design a broken db fixture at the moment